www/js/search.js

```diff
@@ -351,11 +469,19 @@ module.exports = {
 
   // eslint-disable-next-line no-unused-vars
   search(e) {
-    const searchQuery = this.$search.value;
+    let searchType = this.searchType;
+    let searchQuery;
+    if (e === 4) {
```

Comment:
can remove the `// eslint-disable-next-line no-unused-vars ` on line 470 since we are using the var now.